### PR TITLE
GO-1998: add import date to root collection name

### PR DESCRIPTION
**Pull request by AnastasiaShemyakinskaya**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### Description

Currently common collection with imported files looks like this "Notion Import", "Markdown Import" and etc.
I add date for every new created import collection, so now it looks like this "Notion Import 23.10.2023 16:34:00"

### What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Related Tickets & Documents

https://linear.app/anytype/issue/GO-1998/markdown-import-collection-drops-relation-in-all-view-after-importing

### Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [x] 🙅 no documentation needed

### CODE REVIEW

File: core/block/import/converter/collection.go [hunk 1]
```diff
@@ -16,6 +19,8 @@ import (
 	"github.com/anyproto/anytype-heart/util/pbtypes"
 )
 
+const importDateLayout = "02.01.2006 15:04:05"
```
Comment (requilence) [inline core/block/import/converter/collection.go:22]: better to be 2 Jan 2006, because US folks swap month/dat like and it will be ambiguous 01.02.2006

Comment (deff7) [reply core/block/import/converter/collection.go:22]: so maybe use format from locale?

Comment (AnastasiaShemyakinskaya) [reply core/block/import/converter/collection.go:22]: Well we have different languages in Anytype. So maybe `2 Jan 2006` is not suitable for non-english users

Comment (requilence) [reply core/block/import/converter/collection.go:22]: we can use RFC3339, in this case even the sort by name will be consistent :-)

Comment (fat-fellow) [reply core/block/import/converter/collection.go:22]: I also support RFC3339 for the same reasons

Comment (AnastasiaShemyakinskaya) [reply core/block/import/converter/collection.go:22]: IMHO, RFC3339 looks not really user-friendly) But ok

Comment (AnastasiaShemyakinskaya) [reply core/block/import/converter/collection.go]: @requilence please check out

**Review by KirillSto — APPROVED** [review]

**Pull request merged**